server/protocols/webadmin/webadmin-core/src/test/java/org/apache/james/webadmin/routes/HealthCheckRoutesTest.java

```diff
@@ -132,4 +139,24 @@ public void validateHealthchecksShouldReturnInternalErrorWhenOneHealthCheckIsDeg
         .then()
             .statusCode(HttpStatus.INTERNAL_SERVER_ERROR_500);
     }
+
+    @Test
+    public void getHealthchecksShouldReturnEmptyWhenNoHealthChecks() {
+        when()
+                .get(HealthCheckRoutes.CHECKS)
+                .then()
```

Comment:
We tend to indent `.then` on the same level than `when()` to easily read the REST query structure